samples/Catalog/android/build.gradle

```diff
@@ -15,10 +15,10 @@ buildscript {
 allprojects {
     repositories {
         mavenLocal()
-        jcenter()
         maven {
             url 'https://maven.google.com'
         }
+        jcenter()
```

Comment:
I changed the order because otherwise the dependencies on the support library wouldn't resolve correctly.